apps/website/src/server/notifications.ts

```diff
@@ -200,6 +202,26 @@ async function createDiscordNotifications({
     default:
   }
 
+  const linkify = new LinkifyIt();
+
+  const matches = linkify.match(message);
+
+  let offset = 0;
+  if (matches != null) {
+    for (const match of matches) {
+      const linkStart = match.index + offset;
+      const linkEnd = match.lastIndex + offset;
+
+      const linkified = `<${match.text}>`;
+      message =
+        message.slice(0, linkStart) + linkified + message.slice(linkEnd + 1);
+
+      offset += linkified.length - (match.lastIndex - match.index + 1);
+    }
+  }
+
+  message = escapeLinksForDiscord(message);
```

Comment:
that should be able to go now (also the `import LinkifyIt from "linkify-it";`)

```suggestion
  message = escapeLinksForDiscord(message);
```